src/metadata/trackmetadatataglib.cpp

```diff
@@ -148,22 +152,22 @@ inline QString toQString(const TagLib::ID3v2::Frame& frame) {
 }
 
 // Returns the first frame of an ID3v2 tag as a string.
-inline QString toQStringFirst(const TagLib::ID3v2::FrameList& frameList) {
-    if (frameList.isEmpty() || (NULL == frameList.front())) {
-        return QString();
-    } else {
-        return toQString(*frameList.front());
+QString toQStringFirstNotEmpty(
+        const TagLib::ID3v2::FrameList& frameList) {
```

Comment:
I think I can remember, that we had similar discussion before, but I cannot remember the details. 
(Who cares, we can discuss this new, with the new data on this new PR. :-P )

We have a string list, when there are more than one tags of a requested type in the file, right?

We should be sure about the reasons and why we skip a valid empty value. 

Does it happen when a file has two sets of meta-data? I can think of a situation when one application writes the first set and a second application writes the second. I this case, we may want to display the metadata from on application only.

What are your use cases? What are the realistic reasons for string-lists?